src/settings/_box-style.scss

```diff
@@ -3,3 +3,7 @@
 
 // Border radius
 $bdrs-m: 2px !default;
+
+// Form focus
+$bxsh-input-focus: 0 0 3px 0 #60acd0 !default;
+$bdrs-input-focus: solid 1px #22749a;
```

Comment:
Where does this color come from?

Reply:
found on zeplin

Reply:
at least create a default variable for it, please